games-misc/an/an-1.2.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="A fast anagram generator for alphabetic languages"
+DESCRIPTION="an is a fast anagram generator that uses the system's built-in \
+word dictionary to form anagrams. an supports Unicode and theoretically should \
+work for any alphabetic language."
+HOMEPAGE="https://salsa.debian.org/pm/an"
+COPYRIGHT="2012-2014 Paul Martin"
+LICENSE="GNU GPL v2"
+REVISION="1"
+# cloning through git doesn't seem to work nor does downloading an archive
+# from salsa.debian.org, so use the Debian archives instead
+#SOURCE_URI="https://salsa.debian.org/pm/-/archive/master/an-master.tar.gz"
```

Comment:
This could do the trick: https://salsa.debian.org/pm/an/-/archive/403d9a4fd75a5cfae9d5388c4a0e7f889145177f.tar.gz